### PR TITLE
packit: Enable CentOS Stream 10 tests

### PR DESCRIPTION
**Pull request by martinpitt**
The main difficulty here was working around https://issues.redhat.com/browse/TFT-2564

### CODE REVIEW

File: test/browser/browser.sh [hunk 1]
```diff
@@ -46,6 +46,17 @@ if grep -q 'ID=.*rhel' /etc/os-release; then
...
+# HACK: RHEL has these bundled in cockpit-system, but Fedora doesn't; Provides: break in CentOS/RHEL 10
+# due to https://issues.redhat.com/browse/TFT-2564
+if ! grep -q platform:el /etc/os-release; then
+    if [ "$PLAN" = "basic" ]; then
+        dnf install -y cockpit-kdump cockpit-networkmanager cockpit-sosreport
+    fi
+    if [ "$PLAN" = "network" ]; then
+        dnf install -y cockpit-networkmanager
+    fi
+fi
```
Comment (martinpitt) [inline test/browser/browser.sh:58]: This [log shows](https://artifacts.dev.testing-farm.io/65bede2d-b32c-4927-992c-810f7c3d6e73/) that we don't *strictly* need this -- the recommends of "cockpit" are enough. But we've been burned by relying on that before, so for now keep this. We can drop it again once EPEL 10 exists.

**Comment by martinpitt** [comment]
This [testClientCertAuthentication failure](https://artifacts.dev.testing-farm.io/86292385-38d5-4fa7-a5c4-7e1ef1fa1fee/) is a tricky one -- not obvious what happens, and it can't be debugged locally, as testcloud doesn't yet know about "CentOS-Stream-10" image. I'll just ignore it for now.

File: test/browser/browser.sh [hunk 2]
```diff
@@ -46,6 +46,17 @@ if grep -q 'ID=.*rhel' /etc/os-release; then
     dnf install -y kpatch kpatch-dnf
 fi
 
+# HACK: RHEL has these bundled in cockpit-system, but Fedora doesn't; Provides: break in CentOS/RHEL 10
+# due to https://issues.redhat.com/browse/TFT-2564
+if ! grep -q platform:el /etc/os-release; then
```
Comment (jelly) [inline test/browser/browser.sh:51]: Should this also not be el10?

Comment (jelly) [reply test/browser/browser.sh:51]: Nevermind, didn't see the changes below, but then we can drop the `platform:el` grep no? As we have to do it for every image?

Comment (martinpitt) [reply test/browser/browser.sh:51]: This is quite deliberate. *All* RHEL versions bundle the modules in cockpit-system, so we must not try to `dnf install cockpit-kdump` on RHEL 8/9 either.

Well, "must", it'll be idempotent and redundant, but conceptually it's cleaner that way and matches the comment.

**Review by jelly — APPROVED** [review]

**Pull request merged**